DownloadFiles.md

```diff
@@ -0,0 +1,79 @@
+# Download Document Files
+Extensions can download the files associated with documents.
+
+A UIX can identify the file to download in the following ways:
+- Download a file based on the file ID (9.6.002)
+- Download a content file based on the content file type ranking (9.6.002)
+- Download a page file based on the page file number (9.6.002)
+- Download a page file as a .jpg or .png image (9.7.000)
+
+## Usage notes
+Note the following:
+- This endpoint allows users to download files only through a UIX. Users cannot download files by calling the endpoint directly.
+- A user can download a file though a UIX even if the security for the **Document - Download** option on the Case Home > Security > Features page in Ringtail is set to Deny for the user's group.
+- A user cannot download a file associated with a document if the user does not have access to the document, or if the document contains branded redactions.
+- Images are returned at 100% scale. This may result in large images being returned, particularly for documents that were scanned at a high DPI (dots per inch).
+
+## Request format
+### Endpoint
+`POST /api/download`
```

Comment:
The URL will actually be something like `/Ringtail-Svc-Portal/api/download`, but unfortunately the first segment of the path varies based on how the app is installed.

The most reliable way to get the URL is to take `Ringtail.Context.apiUrl` and replace `query` with `download`. Not pretty, need a better solution for this.